config/locales/doc_auth/fr.yml

```diff
@@ -72,10 +72,15 @@ fr:
       card_type: Réessayez avec votre permis de conduire ou carte d’identité délivrée
         par l’État.
       doc:
-        doc_type_check: Les autres pièces d’identité ne sont pas acceptées.
-        wrong_id_type: Nous n’acceptons que les permis de conduire ou les cartes
-          d’identité délivrées par l’État. Les autres pièces d’identité ne sont
-          pas acceptées.
+        doc_type_check: Votre permis de conduire ou votre carte d’identité doit être
+          établi par un État ou un territoire des États-Unis. Nous n’acceptons
+          pas les cartes d’identité militaires, les passeports américains ou
+          autres documents d’identification.
+        wrong_id_type_html: Nous n’acceptons qu’un permis de conduire ou une carte
+          d’identité délivrée par un état ou un territoire des États-Unis. Nous
+          n’acceptons pas les cartes d’identité militaires, les passeports
+          américains ou autres documents d’identification.<a>En savoir plus sur
```

Comment:
<img width="203" alt="image" src="https://github.com/18F/identity-idp/assets/12240896/6fbe4998-1f44-4431-814e-0b2f10e0ef13">

Same nit here: Could you add a space between the . and <a>?